flox-bash/tests/package.bats

```diff
@@ -7,46 +7,62 @@ load test_support.bash
 
 @test "flox package sanity check" {
   # directories
-  [ -d $FLOX_PACKAGE/bin ]
-  [ -d $FLOX_PACKAGE/libexec ]
-  [ -d $FLOX_PACKAGE/libexec/flox ]
-  [ -d $FLOX_PACKAGE/etc ]
-  [ -d $FLOX_PACKAGE/etc/flox.zdotdir ]
-  [ -d $FLOX_PACKAGE/lib ]
-  [ -d $FLOX_PACKAGE/share ]
-  [ -d $FLOX_PACKAGE/share/man ]
-  [ -d $FLOX_PACKAGE/share/man/man1 ]
-  [ -d $FLOX_PACKAGE/share/bash-completion ]
-  [ -d $FLOX_PACKAGE/share/bash-completion/completions ]
+  [ -d "$FLOX_PACKAGE/bin" ]
+  [ -d "$FLOX_PACKAGE/libexec" ]
+  [ -d "$FLOX_PACKAGE/libexec/flox" ]
+  [ -d "$FLOX_PACKAGE/etc" ]
+  [ -d "$FLOX_PACKAGE/etc/flox.zdotdir" ]
+  [ -d "$FLOX_PACKAGE/lib" ]
+  [ -d "$FLOX_PACKAGE/share" ]
+  [ -d "$FLOX_PACKAGE/share/man" ]
+  [ -d "$FLOX_PACKAGE/share/man/man1" ]
+  [ -d "$FLOX_PACKAGE/share/bash-completion" ]
+  [ -d "$FLOX_PACKAGE/share/bash-completion/completions" ]
   # executables
-  [ -x $FLOX_CLI ]
-  [ -x $FLOX_PACKAGE/libexec/flox/gh ]
-  [ -x $FLOX_PACKAGE/libexec/flox/nix ]
-  [ -x $FLOX_PACKAGE/libexec/flox/flox ]
+  [ -x "$FLOX_CLI" ]
+  [ -x "$FLOX_PACKAGE/libexec/flox/gh" ]
+  [ -x "$FLOX_PACKAGE/libexec/flox/nix" ]
+  [ -x "$FLOX_PACKAGE/libexec/flox/flox" ]
   # Could go on ...
 }
 
 @test "flox --prefix" {
-  run $FLOX_CLI --prefix
+  run "$FLOX_CLI" --prefix
   assert_success
-  assert_output $FLOX_PACKAGE
+  assert_output "$FLOX_PACKAGE"
 }
 
 @test "flox install by /nix/store path" {
-  run $FLOX_CLI install -e $TEST_ENVIRONMENT $FLOX_PACKAGE
+  run "$FLOX_CLI" install -e "$TEST_ENVIRONMENT" "$FLOX_PACKAGE"
   assert_success
   assert_output --partial "Installed '$FLOX_PACKAGE' package(s) into '$TEST_ENVIRONMENT' environment."
 }
 
 @test "flox list after installing by store path should contain package" {
-  run $FLOX_CLI list -e $TEST_ENVIRONMENT
+  run "$FLOX_CLI" list -e "$TEST_ENVIRONMENT"
   assert_success
   assert_output --partial "Curr Gen  1"
   assert_output --partial "0  $FLOX_PACKAGE  $FLOX_PACKAGE_FIRST8"
 }
 
+@test "flox activate on multiple environments" {
+  run "$FLOX_CLI" create -e "${TEST_ENVIRONMENT}-2"
+  assert_success
+  run "$FLOX_CLI" install -e "${TEST_ENVIRONMENT}-2" "$FLOX_PACKAGE"
+  assert_success
+  (
+    run "$FLOX_CLI" activate -e "$TEST_ENVIRONMENT" -e "${TEST_ENVIRONMENT}-2" \
```

Comment:
I'm seeing a:
```
     INFO environment not found: /var/folders/2w/r9qm37r12qs4xn3sjn4cv2ym0000gn/T/nix-shell.MBRxh3/tmp.y9ubrRGEzs/.local/share/flox/environments/local/aarch64-darwin._testing_
```
Is that expected?

Reply:
No that's not normal. Take it for another spin with the latest changes?

Reply:
try it now

Reply:
That's a bit too relaxed as I can re-order `_testing_` and `_testing_-2` and it passes either way

Still seeing the INFO when I force the regex to fail

Reply:
alright if the next one doesn't work then I'm running out of ideas lol

Reply:
That fixed it!